Makefile

```diff
@@ -95,7 +96,10 @@ local: validate test install
 ci-pr: resolve validate build test integration-test
 
 .PHONY: ci-main
-ci-main: resolve validate build test integration-test upload-binaries
+ci-main: resolve validate build test integration-test
+
+.PHONY: ci-publish
+ci-publish: resolve validate build test integration-test upload-binaries
```

Comment:
With the new UNSTABLE preset, there is no need to introduce a dedicated target ci-publish

Reply:
Thanks. I changed it.